### DISCUSSION TITLE
Welcome to DnetIndexedDb Discussions!

### DISCUSSION THREAD

**Discussion by amuste** [amuste/DnetIndexedDb#10] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by Kylar182** [comment]
What was added /fixed updated with 2.1? Are there any other Native IndexedDB methods that we could integrate or do you have everything integrated? Would you like help with the project at all? I'm terrible at JavaScript but I can show some more C# examples, particularly using T Typed Local Query Repositories like Dotnet Core Backend setups.

**Comment by amuste** [comment]
Hi @Kilar182. Sorry for the late reply.

What was added /fixed updated with 2.1?
Not need to send null in key property when using in-line-keys with Key generator

Are there any other Native IndexedDB methods that we could integrate or do you have everything integrated?
I have been reviewing the standard documentation. At the moment I think this is what is most used, but surely something will be missing. If you identify any missing functionality please notify me.

Would you like help with the project at all?
Of course. Any help from the community is always welcome.

I'm terrible at JavaScript but I can show some more C# examples, particularly using T Typed Local Query Repositories like Dotnet Core Backend setups.
Yes, that would be great.

And thank you very much for your interest and support.
